Added clipboard support to tutorial's code snippets

bmispelon opened the pull request:
This works in tandem with django/djangoproject.com#678 to add a "copy to clipboard" icon on the tutorial's code snippets (only those with a filename for now).

erikr commented:
Looks good to me! :shipit:

bmispelon commented:
As discussed on django/djangoproject.com#678, the feature is now enabled on all code snippets (that is, code blocks that have a filename header).

timgraham commented on docs/_ext/djangodocs.py:
```diff
@@ -109,8 +109,9 @@ def warner(msg):
+    css_classes = ['highlight-%s' % lang, 'clippify']
     starttag = self.starttag(node, 'div', suffix='',
-                             CLASS='highlight-%s' % lang)
+                             CLASS=' '.join(css_classes))
```
Was a previous iteration of the code more complicated? Not sure the intermeidate `css_classes` variable has much benefit.

timgraham commented on docs/_ext/djangodocs.py:
```diff
@@ -109,8 +109,9 @@ def warner(msg):
                                                    warn=warner,
                                                    linenos=linenos,
                                                    **highlight_args)
+    css_classes = ['highlight-%s' % lang, 'clippify']
```
I'm inclined to use a more generic CSS class name like "snippet" that might be useful for other purposes. What do you think?

bmispelon commented:
I reverted to the previous way of generating the `CSS_CLASS` and changed the class name to the more generic `snippet` (we're already using `snippet-filename` so that makes sense).

I also updated the joint pull request on djangoproject.com to reflect this new class name.
